csv-samples/readme.md

```diff
@@ -0,0 +1,64 @@
+# CSV Parser Node Samples
+## General instructions
+This directory contains examples of the use of the CSV Parser node in App Connect integrations.
+Each example comes with a brief description of the scenario covered, and a flow file that implements the scenario.
+
+You'll find the following resources useful for all of them
+* [How to use the CSV Parser Node](https://developer.ibm.com/integration/blog/2017/10/18/how-to-use-the-csv-parser-node-in-app-connect/)
+* [How to upload integration flow files](https://developer.ibm.com/integration/blog/2017/09/29/exporting-importing-flows-ibm-app-connect/)
+
+## Use-cases
+### CSV file from Box used to add/update contacts to Salesforce
+#### Overview
+In this scenario, an App Connect flow exposes an API that
+allows you to identify a file in box via its file URL.
+The file contains a table of 'contact' details, which is used to
+update contact information in salesforce.
+
+#### To-do list
+1. Upload your CSV file to Box
+  * You can use `samplecontacts.csv` to get started
+2. Upload the flow file `CSV-File-in-Box-to-Salesforce-contacts.yaml` (see *Resources* below)
+  * This will create a new integration, which you will see in the App Connect dashboard
+3. Open the newly created application
+  * Review the configuration of each node, so that you understand how it has been configured
+  * If you do not already have a connection to Box - Click on the Box node, and connect to Box
+  * If you do not already have a connection to Salesforce - Click on the Salesforce node, and connect to Salesforce
+4. Save and start the integration
+5. Click on the 'manage' tab for your API, and set up the API authentication
+  * Near the bottom of the page, you'll see a link to a dedicated 'API Portal' for consumers of your API to use
+6. View your file on Box - and copy the file URL from the browser
+7. Open up the API Portal for your API, and use the UI to 'try out' invoking your new API
+  * Copy the file URL from box into the API input shown
```

Comment:
For example:
**data**
```
{
  "fileurl": "https://app.box.com/file/123456789"
}
```
* Click 'Call operation'
  * Under **Response**, you should see 'Code: 201 Created'
  * In the Response body, you should see the fileURL parameter used
  * If you Switch to your Salesforce account, you should see two new Contacts created from the CSV file:
```
NAME
Fred Jones
Joe Smith
```

Reply:
Done